cmd/pmm-managed-starlark/main_test.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/percona/pmm-managed/services/checks"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRunChecks(t *testing.T) {
+	l := logrus.WithField("component", "pmm-managed-starlark")
+
+	testCases := []struct {
+		data   checks.StarlarkScriptData
+		err    bool
+		panics bool
+	}{
+		{
+			data: checks.StarlarkScriptData{
+				CheckName:         "invalid version",
+				Script:            "def check(): return []",
+				CheckVersion:      5,
+				QueryActionResult: "some result",
+			},
+			err:    true,
+			panics: false,
+		},
+		{
+			data: checks.StarlarkScriptData{
+				CheckName:         "invalid starlark syntax",
+				Script:            "@ + @",
+				CheckVersion:      1,
+				QueryActionResult: "some result",
+			},
+			err:    true,
+			panics: false,
+		},
+		{
+			data: checks.StarlarkScriptData{
+				CheckName:         "invalid query action result",
+				Script:            "def check(): return []",
+				CheckVersion:      1,
+				QueryActionResult: "some result",
+			},
+			err:    true,
+			panics: false,
+		},
+		{
+			data: checks.StarlarkScriptData{
```

Comment:
for some reason this test case fails, even though it is supposed to panic, it fails the test, I can't figure out why. PTAL @AlekSi @artemgavrilov

Reply:
Ok, but we don't have a code that panics, so why it should panic?

Reply:
this case will run this script `"def check(rows): return [1] * (1 << 30-1)"` which will fail with `runtime error: out of memory` and a traceback, isn't that a panic?

Reply:
Nope, that unrecoverable runtime error, not a panic

Reply:
i don't think I can test that then, will remove this case